src/scones.md

```diff
@@ -0,0 +1,28 @@
+# Scones
+
+![scones](pix/scones.webp "Scones")
+
+- ⏲️ Prep time: 10 min
+- 🍳 Cook time: 12 min
+- 🍽️ Servings: 12 pieces
+
+## Ingredients
+
+-  3/4 cups of cream
+-  1 cup of milk
+-  3 tablespoons of icing sugar
+-  3 cups of self-rising flour, sifted
+
+## Directions
+
+1. With a rotary beater, beat together cream, milk and icing sugar for 1 minute.
+2. Add the flour and combine
+3. Press onto a flat surface and cut into desired shapes
```

Comment:
Into circles. Tip: do not rotate the cutter as it will prevent rising of the scone [source](https://youtu.be/KAbkXMOuCcY?t=1139)